application.py

```diff
@@ -1,32 +1,96 @@
-from flask import Flask
 
-# print a nice greeting.
-def say_hello(username = "World"):
-    return '<p>Hello %s!</p>\n' % username
+# coding: utf-8
+
+# In[ ]:
+
+
+from flask import Flask, redirect, url_for, session, request, jsonify
+from requests_oauthlib import OAuth2Session
+from requests_oauthlib.compliance_fixes import linkedin_compliance_fix
+import os 
+
+#for SSL
+os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+
+
+# In[ ]:
+
+
 
 # some bits of text for the page.
 header_text = '''
```

Comment:
i suggest getting rid of our names on the page, leave the login button, and make the route for index page using decorators.